docs/dev/build_linux.md

```diff
@@ -76,6 +76,7 @@ You can use the following additional build options:
      ```
      export PYTHONPATH=<openvino_repo>/bin/intel64/Release/python:<openvino_repo>/tools/ovc:$PYTHONPATH
      export LD_LIBRARY_PATH=<openvino_repo>/bin/intel64/Release:$LD_LIBRARY_PATH
+     export PATH=<openvino_repo>/tools/ovc/openvino/tools/ovc:$PATH
```

Comment:
how to call this ovc tool then? ovc.py ?

Reply:
Just `ovc` works well

Reply:
I don't see `ovc` file here https://github.com/openvinotoolkit/openvino/tree/master/tools/ovc/openvino/tools/ovc

Reply:
![image](https://github.com/user-attachments/assets/5f9eb756-93e0-4176-869d-1ef51eb9b328)

ovc.py presents